assets/src/components/activities/adaptive/components/authoring/LayoutEditor.tsx

```diff
@@ -401,7 +401,7 @@ const LayoutEditor: React.FC<LayoutEditorProps> = (props) => {
       }
     },
     ['Delete', 'Backspace'],
-    {},
+    { ctrlKey: true },
     [selectedPartAndCapabilities, configurePartId],
```

Comment:
This one is already approved as part of https://github.com/Simon-Initiative/oli-torus/pull/4861 PR . I will delete that PR.